client/src/App.vue

```diff
@@ -85,13 +95,20 @@ export default {
           EventBus.$emit(PAGE_DATA_REFRESH_EVENT)
         })
         .catch(err => {
-          alert(err + '\nSystem Error')
           StoreHelper.setLoading(null, false)
           this.$router.push('/')
+          setApiError(err + '\n. System Error')
```

Comment:
Check that setApiError works with this newline. Does it convert the new line into an html break?

Reply:
It doesn't. I will be removing this newline for now. Thank you for your observation!